Code/GraphMol/Atropisomers.cpp

```diff
@@ -669,8 +675,8 @@ bool WedgeBondFromAtropisomerOneBond2d(
     }
   }
 
-  if (bestBondEnd >= 0)  // we found a good one
-  {
+  if (bestBondEnd >= 0) {
```

Comment:
Should there be an error if no bonds were in a ring?

Reply:
nope. The atropisomer code allows non-ring bonds.